packages/runtime/src/types/application.ts

```diff
@@ -20,17 +20,21 @@ export type GridCallbacks = {
 
 export type ComponentParamsFromApp = {
   gridCallbacks?: GridCallbacks;
+  hooks?: AppHooks;
 };
 
 export type AppProps = {
   options: Application;
   services: UIServices;
   debugStore?: boolean;
   debugEvent?: boolean;
-  hooks?: AppHooks;
 } & ComponentParamsFromApp;
 
 export type AppHooks = {
+  // after app first render
   didMount?: () => void;
+  // app updates after schema changes
   didUpdate?: () => void;
+  // app updates after html elements change
+  didHtmlUpdate?: () => void;
```

Comment:
maybe naming like this?

didUpdate -> willUpdate
didHtmlUpdate -> didUpdate

or
didHtmlUpdate -> didDomUpdate

Reply:
OK, I changed to `didDomUpdate`.